lib/Dispatcher.js

```diff
@@ -147,6 +147,10 @@ module.exports = function () {
      * @throws {Error} if store has handler registered that does not exist
      */
     Dispatcher.prototype.dispatch = function dispatch(actionName, payload) {
+        if (actionName === undefined) {
```

Comment:
To answer your question, I think we may as well check any falsy value here: `if (!actionName)` and then the message can just say that the `actionName` is invalid.